### DISCUSSION TITLE
[@types/react] needs support for Popover API

### DISCUSSION THREAD

**Discussion by viveleroi** [DefinitelyTyped/DefinitelyTyped#66550] [Issues with a @types package] [answered]
The native Popover API is already supported in Chrome/Edge and behind flags in Firefox. It appears like this is not yet supported in @types/react: `<div popover>text</div>` or `htmlElementRef.togglePopover()` etc.

`Property 'popover' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'`

React version: 18.2.0
@types/react version: 18.2.21
@types/react-dom version: 18.2.7
typescript version: 5.2.2

It *works* in the browser when I save and run the code it, so it appears to be just a type issue. I don't know where I'm really supposed to report this so I'm starting here.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
I am on it.

Looks like need to add support to React first - https://github.com/facebook/react/issues/27479

**Comment by acusti** [comment]
@Semigradsky thanks for opening [the PR](https://github.com/facebook/react/pull/27480) in react to add support for the popover API! 🤞 the issue with the fixtures is resolved soon.

is this the best place to keep an eye out for typescript adding the types to its react libdef, or is there a proper GitHub issue in some repo somewhere i should subscribe to?

**Comment by mkrause** [comment]
React support landed: https://github.com/facebook/react/pull/27981

**Comment by eps1lon** [comment, marked as answer]
Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69670. Thank you for the report.
